specification/cosmos-db/resource-manager/readme.md

```diff
@@ -27,10 +27,26 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-2022-11
```

Comment:
removing this will make SDK generation failed. I guess you may want to set tag as `2022-11-15-preview`?

Reply:
Added tag: package-preview-2022-11